src/certificate/Certificate.test.tsx

```diff
@@ -16,9 +16,10 @@ jest.mock("@logion/node-api/dist/LogionLoc");
 jest.mock("../common/api");
 jest.mock("../common/hash");
 jest.mock("../common/Model");
+jest.mock('@logion/extension');
```

Comment:
This is implicit. You may safely remove this line.

Reply:
OK, thanks !. Will be done in the next PR.